modules/nextflow/src/main/groovy/nextflow/trace/WebLogObserver.groovy

```diff
@@ -113,15 +138,22 @@ class WebLogObserver implements TraceObserver{
         // This is either set by the user or via Nextflow name generator
         runName = session.getRunName()
         runId = session.getUniqueId()
-        asyncHttpMessage("started")
+        workFlowMetadata = session.binding.getVariable('workflow') as WorkflowMetadata
+
+        def fullMetadata = [
+                "params": session.binding.getProperty('params'),
+                "workflow": workFlowMetadata
+        ]
+
```

Comment:
I would propose to use a small value object with just these two fields. it can be defined as a static inner class maybe something like `FlowMetadata` or `FlowPayload`.